### PR TITLE
Test against PHP 7

### PR DESCRIPTION
**Pull request by WouterJ**

### CODE REVIEW

**Comment by dbu** [comment]
there seems to be a problem with annotations reading: https://travis-ci.org/symfony-cmf/standard-edition/jobs/81148240 (or maybe the `@Field` thing and a weird error report)

**Comment by WouterJ** [comment]
Problem is not up to date deps, updated the lock file now

**Comment by dbu** [comment]
restarted the php 7 tests as yesterday the AWS outage seems to have affected this. now we get a fatal about `@String` so i guess we miss some conversion to `@Field`.

File: composer.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
         "symfony/assetic-bundle": "~2.3",
         "symfony/swiftmailer-bundle": "~2.3",
         "symfony/monolog-bundle": "~2.4",
-        "symfony-cmf/symfony-cmf": "1.2.*",
+        "symfony-cmf/symfony-cmf": "1.3.*",
```
Comment (lsmith77) [inline composer.json]: I guess this requires https://github.com/symfony-cmf/symfony-cmf/pull/222 to be merged ?

**Comment by lsmith77** [comment]
ok .. with another update DoctrinePHPCRBundle should be fine .. but I am not investigating an issue with fixture loading in ``DoctrineManager:transactional()``

**Comment by WouterJ** [comment]
Let's see

**Comment by WouterJ** [comment]
I'm sorry, a newer has been introduced again: 

 > [Symfony\Component\DependencyInjection\Exception\ServiceNotFoundException]   
 > You have requested a non-existent service "doctrine_phpcr.admin.jackalope.repository.default".

**Comment by lsmith77** [comment]
that one is fixed by https://github.com/doctrine/DoctrinePHPCRBundle/pull/215 .. which in turn I have not merged since its showing new deprecation notices.

the actual issue I am not facing is with fixture loading as noted above.

**Comment by lsmith77** [comment]
merged the DoctrinePHPCRBundle branch and updated the dependencies to illustrate the issue with ``DoctrineManager:transactional()`` and fixture loading.

**Comment by lsmith77** [comment]
tried it also on mysql .. same issue than on sqlite

**Comment by lsmith77** [comment]
@Ocramius did you ever run into issues with Fixture loading with the ``transactional()`` method with the ORM?

**Comment by Ocramius** [comment]
@lsmith77 never had any issue with that

**Comment by lsmith77** [comment]
FYI .. since we do not seem to be able to figure out what is going wrong with ``transactional()`` we will revert the feature for now and potentially re-add it later.

**Comment by lsmith77** [comment]
can you update the composer.lock now that ``DocumentManager::transactional()`` has been removed?

**Comment by WouterJ** [comment]
Tests pass now. Squashed the commits & ready to merge

**Pull request merged**